packages/sql-ts/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "sql-ts",
```

Comment:
Let's add `"private": true` so npm doesn't try to publish this

Reply:
TIL that's what the meant...